tests/ui/consts/miri_unleashed/mutable_references.stderr

```diff
@@ -1,8 +1,13 @@
-error: encountered mutable pointer in final value of static
+error[E0080]: it is undefined behavior to use this value
   --> $DIR/mutable_references.rs:7:1
    |
 LL | static FOO: &&mut u32 = &&mut 42;
-   | ^^^^^^^^^^^^^^^^^^^^^
+   | ^^^^^^^^^^^^^^^^^^^^^ constructing invalid value at .<deref>: encountered mutable reference or box pointing to read-only memory
```

Comment:
Okay nice, it shows says where in the value the error comes from. :)

It is probably confusing why there is any immutable memory here, without an explanation like "implicitly created memory for the tail expression of a static is always read-only" or so. But then, this error can't even be triggered without miri-unleashed... so it's not worth a lot of effort to make nicer errors.